parfive/downloader.py

```diff
@@ -534,7 +547,7 @@ def callback(token, future, main_pb):
         """
         parse = urllib.parse.urlparse(url)
         try:
-            async with aioftp.ClientSession(parse.hostname, **kwargs) as client:
+            async with aioftp.ClientSession(parse.hostname, headers=self.headers, **kwargs) as client:
```

Comment:
The headers have to be passed to the `session.get` call not the `ClientSession` constructor. https://aiohttp.readthedocs.io/en/stable/client_advanced.html#custom-cookies

Reply:
It is mentioned that,
`You also can set default headers for all session requests:`
`headers={"Authorization": "Basic bG9naW46cGFzcw=="}`
`async with aiohttp.ClientSession(headers=headers) as session:`

So this should not throw an error, right?

Reply:
huh ok, I missed that. Yeah seems like it should work then.